### PR TITLE
Link to bundling docs in Readme

### PR DESCRIPTION
**Pull request by JonathanCrd**
Solves https://github.com/Azure/autorest.typescript/issues/1214.

This PR targets v6. Added a new section in the Readme referencing the JavaScript bundle as seen in [this file](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/storage/storage-blob/README.md#javascript-bundle).

### CODE REVIEW

**Review by joheredi — APPROVED** [review]

**Review by sarangan12 — APPROVED** [review]

**Comment by ramya-rao-a** [comment]
@JonathanCrd Looks like the `check git tree` step in the tests are failing

File: test/smoke/generated/web-resource-manager/package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "rollup-plugin-sourcemaps": "^0.4.2",
     "typescript": "~4.2.0",
     "uglify-js": "^3.4.9",
-    "@azure/identity": "^2.0.1",
+    "@azure/identity": "2.0.0-beta.6",
```
Comment (joheredi) [inline test/smoke/generated/web-resource-manager/package.json:41]: Why are we going back to beta? 2.0.1 is newer, isn't it?

**Comment by joheredi** [comment]
> @JonathanCrd Looks like the `check git tree` step in the tests are failing

@JonathanCrd you may be missing `npm run build` before re-generating  tests and smoke-tests

**Comment by JonathanCrd** [comment]
@joheredi you're right, I'm regenerating the smoke tests again

**Pull request merged**